README.md

```diff
@@ -22,7 +22,24 @@ The full reference documentation for rules is at https://haskell.build.
 
 ## Setup
 
-You'll need [Bazel >= 0.27][bazel-getting-started] installed.
+You'll need [Bazel >= 0.29][bazel-getting-started] installed.
+
+If you are on NixOS, skip to the [Nixpkgs](#Nixpkgs) section.
+
+### System dependencies
+
+On a Unix system you will need the following tools installed.
+
+- `gcc`
```

Comment:
On Ubuntu, I ended up with the following packages:

`build-essential libtinfo5 libtinfo-dev python libgmp-dev libffi-dev`

Not included are the ones silently pulled in by the base distribution, obviously.

Reply:
Thanks, I've added a line with the Ubuntu packages.